### DISCUSSION TITLE
Training and testing RMSE energy drops significantly at training step=decay steps

### DISCUSSION THREAD

**Discussion by hadi-bourji** [deepmodeling/deepmd-kit#1509] [Q&A] [answered]
I have run several trainings with the same decay step value and once the training gets past the point of the decay steps value the error drops significantly. Is there any reasoning behind this? I had assumed once it gets to decay steps the only significance is that the learning rate reaches its minimum value

**Comment by njzjz** [comment, marked as answer]
The total loss is a function of the learning rate.

**Reply by hadi-bourji** [reply]
Right, but the RMSE energy is only a function of the predicted and actual energies correct? I am confused why the learning rate would affect the RMSE

**Reply by njzjz** [reply]
If the learning rate is too high, the learning will jump back and forth over a minimum. You can read _[Goodfellow et al, Deep Learning (2016), MIT press, pp 424](https://www.deeplearningbook.org/contents/guidelines.html)_.
